operator/builtin/input/file/file_test.go

```diff
@@ -1000,7 +1122,7 @@ LOOP:
 		select {
 		case e := <-c:
 			receivedMessages = append(receivedMessages, e.Record.(string))
-		case <-time.After(500 * time.Millisecond):
+		case <-time.After(time.Second):
```

Comment:
Was this increased for a reason?

Reply:
I think I saw a couple failures on this and increased it. I'll change it back and we'll see what happens.